src/main/java/net/mcreator/element/types/Armor.java

```diff
@@ -163,6 +179,74 @@ public Model getBootsModel() {
 		return Model.getModelByParams(getModElement().getWorkspace(), bootsModelName, modelType);
 	}
 
+	public Model getHelmetItemJSONModel() {

```

Comment:
I'd rename these methods to `getHelmetItemModel()` etc, as the current name is misleading

Reply:
How is it misleading? It is directly referring to the JSON model of the item.
(I changed it. I'm just curious)

Reply:
The Model can be of any type, not just JSON. The new name is also consistent with other elements